tests/simple/testdata/fields.textproto

```diff
@@ -353,7 +353,7 @@ section {
   }
   test {
     name: "map_value_mix_type"
-    expr: '{"map": {"k": "v"}, "list": [1]}["map"]'
+    expr: "{\"map\": {\"k\": \"v\"}, \"list\": [1]}[\"map\"]"
```

Comment:
nit: Since we're updating these lines, maybe prefer the single quote inside the string, e.g. `"'map'"` instead of `"\"map\""`